framework/src/org/checkerframework/framework/stub/StubParser.java

```diff
@@ -376,63 +384,68 @@ private void parse(TypeDeclaration typeDecl,
         for (Map.Entry<Element, BodyDeclaration> entry : elementsToDecl.entrySet()) {
             final Element elt = entry.getKey();
             final BodyDeclaration decl = entry.getValue();
-            if (elt.getKind().isField())
+            if (elt.getKind().isField()) {
                 parseField((FieldDeclaration)decl, (VariableElement)elt, atypes, declAnnos);
-            else if (elt.getKind() == ElementKind.CONSTRUCTOR)
+            } else if (elt.getKind() == ElementKind.CONSTRUCTOR) {
                 parseConstructor((ConstructorDeclaration)decl, (ExecutableElement)elt, atypes, declAnnos);
-            else if (elt.getKind() == ElementKind.METHOD)
+            } else if (elt.getKind() == ElementKind.METHOD) {
                 parseMethod((MethodDeclaration)decl, (ExecutableElement)elt, atypes, declAnnos);
-            else { /* do nothing */
+            } else {
+                /* do nothing */
                 stubWarning("StubParser ignoring: " + elt);
             }
         }
     }
 
-    private void parseType(ClassOrInterfaceDeclaration decl, TypeElement elt, Map<Element, AnnotatedTypeMirror> atypes, Map<String, Set<AnnotationMirror>> declAnnos) {
+    private void parseType(ClassOrInterfaceDeclaration decl,
+            TypeElement elt, Map<Element, AnnotatedTypeMirror> atypes,
+            Map<String, Set<AnnotationMirror>> declAnnos) {
         annotateDecl(declAnnos, elt, decl.getAnnotations());
         AnnotatedDeclaredType type = atypeFactory.fromElement(elt);
         annotate(type, decl.getAnnotations());
-        {
```

Comment:
To avoid the extra indentation. There are only three statements after the end of the code block and the benefit in limiting the scope didn't seem to outweigh the negative effect of the extra nesting.